package.json

```diff
@@ -39,8 +39,14 @@
     "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
   },
   "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
-  "dependencies": {},
-  "devDependencies": {},
+  "dependencies": {
```

Comment:
this makes no sense

Reply:
I followed latest webpack installation steps: https://github.com/rails/webpacker#installation